task/gcp/task.go

```diff
@@ -142,6 +148,10 @@ type Task struct {
 
 func (t *Task) Create(ctx context.Context) error {
 	logrus.Info("Creating resources...")
+	logrus.Info("[0/14] Parsing PermissionSet...")
```

Comment:
[still not a thing](https://xkcd.com/163/)

Reply:
🤣 ✖️ 💯